features/support/fs.rs

```diff
@@ -1,9 +1,11 @@
 use tempdir::TempDir;
 use std::io::{self, Write};
-use std::fs::{DirBuilder, File};
+use std::fs::{copy, create_dir_all, DirBuilder, File};
```

Comment:
From what I hear, the general practice is to import the module and call these functions with the module prefixed to add context like so

```rust
fs::copy(...)
```

So just import `self` and prefix the module methods with it.